app/models/form/sales/questions/age2.rb

```diff
@@ -0,0 +1,11 @@
+class Form::Sales::Questions::Age2 < ::Form::Question
+  def initialize(id, hsh, page)
+    super
+    @id = "age2"
+    @check_answer_label = "Buyer 2’s age"
+    @header = "Age"
+    @type = "numeric"
+    @page = page
+    @width = 2
```

Comment:
Should this be 3?

Reply:
The width?

Reply:
Yep? You could feasibly have a 100+ year old buyer?

Reply:
I just mimicked what was set up for the lead tenant age too. I think that one should also be changed too if this one is